app/src/main/java/org/jellyfin/androidtv/model/trailers/lifter/ExternalTrailerLifter.kt

```diff
@@ -1,15 +1,28 @@
 package org.jellyfin.androidtv.model.trailers.lifter
 
-import android.net.Uri
 import org.jellyfin.androidtv.model.trailers.external.ExternalTrailer
 import org.jellyfin.apiclient.model.entities.MediaUrl
+import java.net.MalformedURLException
+import java.net.URL
 
 abstract class ExternalTrailerLifter {
 	private val domainRegex = Regex("""^(.*\.|)(.+\...+)$""")
```

Comment:
I'm not really sure what this regex does, strip subdomains?

Reply:
That was the idea. Should I write test cases for it as well?

Reply:
You can probably do it without the regex

Reply:
Got rid of it